packages/flutter/lib/src/foundation/change_notifier.dart

```diff
@@ -62,11 +62,11 @@ abstract class Listenable {
   /// Return a [Listenable] that triggers when any of the given [Listenable]s
   /// themselves trigger.
   ///
-  /// The list must not be changed after this method has been called. Doing so
-  /// will lead to memory leaks or exceptions.
+  /// [listenables] must not be changed after this method has been called.
```

Comment:
nit: Per style guide, docs should start grammatically correct with a capital letter, see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-correct-grammar.

May have to reword this sentence slightly to make that possible here.